packages/plugins/basic-pipeline/src/index.js

```diff
@@ -28,6 +27,10 @@ class BasicPipeline {
     this.plugins = embark.pluginsAPI;
     this.pipelinePlugins = this.plugins.getPluginsFor('pipeline');
     this.pipelineConfig = embark.config.pipelineConfig;
+
+    const env = embark.config.env;
+    this.webpackConfigName = embark.pluginConfig[env] && embark.pluginConfig[env].webpackConfigName ? embark.pluginConfig[env].webpackConfigName : 'development';
```

Comment:
Eric's version is cooler and shorter
```suggestion
    this.webpackConfigName = embark.pluginConfig[env]?.webpackConfigName ?? 'development';
```

Reply:
I keep forgetting that babel can handle TS code. Updating..